src/internal.c

```diff
@@ -8140,6 +8140,21 @@ static void FreeProcPeerCertArgs(WOLFSSL* ssl, void* pArgs)
     }
 }
 
+#ifdef HAVE_PK_CALLBACKS
+static void InitSigPkCb(WOLFSSL* ssl, struct SignatureCtx* ctx) {
+    if ((ssl != NULL) && (ctx != NULL)) {
+    #ifdef HAVE_ECC
+        ctx->eccVerifyCb = ssl->ctx->EccVerifyCb;
+        ctx->eccVerifyCtx = ssl->ctx->EccVerifyCtx;
```

Comment:
The `ssl->ctx->EccVerifyCtx` should be `ssl->EccVerifyCtx`. Same for `RsaVerifyCtx`.

Reply:
Great, thanks for pointing that out, and updating the PR title.